apps/testnorge-arena/arena-core/src/main/java/no/nav/registre/arena/core/consumer/rs/util/ConsumerUtils.java

```diff
@@ -47,6 +47,28 @@ public RequestEntity createPostRequest(
                 .body(requester);
     }
 
+    public RequestEntity createPostRequest(
```

Comment:
For å slippe duplisert kode her kan du f.eks lage en private-metode:

`
private void opprettRequest(LocalDate startDato, List<RettighetSyntRequest> requester) {
    LocalDate sluttDato = startDato.plusDays(rand.nextInt(365 / 2) + (365 / (long) 2));
    requester.add(RettighetSyntRequest.builder()
        .fraDato(startDato)
        .tilDato(sluttDato)
        .utfall(UTFALL_JA)
        .vedtakTypeKode(VEDTAK_TYPE_KODE_O)
        .vedtakDato(startDato)
        .build());
}
`

som hver av createPostRequest-metode kaller inne i sin løkke etter å ha regnet ut sin startdato?

Reply:
God ide, skal se på det!